pages/reminders/index.js

```diff
@@ -66,7 +67,7 @@ export default function Reminders() {
   } = useSWR(userId ? `/api/user/${userId}/owned` : null);
 
   const groupedReminders = useMemo(
-    () => (reminders ? groupReminders(reminders) : []),
+    () => groupReminders(reminders ?? []),
```

Comment:
here again the code looks different to Dominiks code. Please make sure that you don't overwrite the correct version.